temperature/routers.py

```diff
@@ -0,0 +1,40 @@
+from datetime import datetime
+
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from temperature import crud, schemas
+from database import AsyncSessionLocal
+
+router_temp = APIRouter()
+
+
+async def get_db():
+    async with AsyncSessionLocal() as db:
+        yield db
+
+
+@router_temp.get("/", response_model=list[schemas.Temperature])
+async def read_city(db: AsyncSession = Depends(get_db)):
```

Comment:
The function name `read_city` is misleading in the context of the temperature module. Consider renaming it to something more descriptive, such as `read_temperatures`, to better reflect its purpose.